lib/puppet/provider/service/rcng.rb

```diff
@@ -0,0 +1,50 @@
+Puppet::Type.type(:service).provide :rcng, :parent => :bsd do
+  desc <<-EOT
+    RCng service management with rc.d
+  EOT
+
+  defaultfor :operatingsystem => [:netbsd, :cargos]
```

Comment:
i'm currently unavailable, but ill apple the suggested fixes when im back. 
-- 
jasper